### PR TITLE
added response to list subscriptions inline button

### PR DESCRIPTION
**Pull request by athulkrishnaaei**

### CODE REVIEW

File: src/bot/commands/help_command_info.rs [hunk 1]
```diff
@@ -97,7 +98,9 @@ impl HelpCommandInfo {
             HelpCommand::Help => HELP.to_string(),
             HelpCommand::Subscribe => SUBSCRIBE.to_string(),
             HelpCommand::Unsubscribe => UNSUBSCRIBE.to_string(),
-            HelpCommand::ListSubscriptions => LIST_SUBSCRIPTIONS.to_string(),
+            HelpCommand::ListSubscriptions => {
```
Comment (ayrat555) [inline src/bot/commands/help_command_info.rs:101]: help_command_info is used for showing information. so it shouldn't show actual subscriptions

File: src/bot/commands.rs [hunk 2]
```diff
@@ -275,6 +275,23 @@ pub trait Command {
         telegram_client::api().clone()
     }
 
+    fn list_subscriptions(db_connection: &mut PgConnection, message: &Message) -> String {
```
Comment (ayrat555) [inline src/bot/commands.rs:278]: you should modify the list_subscriptions command. this function is not needed

**Review by ayrat555 — CHANGES_REQUESTED** [review]

File: src/bot/commands.rs (hunk 2 above)
Comment (athulkrishnaaei) [reply src/bot/commands.rs:278]: okay

**Comment by ayrat555** [comment]
already implemented in the master branch

**Pull request closed without merging**